### PR TITLE
[Hub Generated] Review request for Microsoft.VirtualMachineImages to add version preview/2019-05-01-preview

### PR DESCRIPTION
**Pull request by feiyushi**
If you are a MSFT employee you can view your [work branch via this link](https://portal.azure-devex-tools.com/app/branch/feiyushi/azure-rest-api-specs/dev-imagebuilder-Microsoft.VirtualMachineImages-2019-05-01-preview...master).

### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

**Comment by feiyushi** [comment]
cc: @paulmey

File: specification/imagebuilder/resource-manager/readme.go.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ These settings apply only when `--go` is specified on the command line.
 batch:
   - tag: package-2018-02
   - tag: package-2019-02
+  - tag: package-2019-05
```
Comment (jhendrixMSFT) [inline specification/imagebuilder/resource-manager/readme.go.md]: According to the README.md file the tag is `package-preview-2019-05`

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Can you please also fix up the examples listed in the following model validation failures?
https://dev.azure.com/azure-sdk/public/_build/results?buildId=20341

File: specification/imagebuilder/resource-manager/readme.go.md (hunk 1 above)
Comment (feiyushi) [reply specification/imagebuilder/resource-manager/readme.go.md]: thanks. fixed the tag for go sdk

**Comment by jhendrixMSFT** [comment]
Thanks there are just two model validation failures remaining, see https://travis-ci.org/Azure/azure-rest-api-specs/jobs/523230496

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**